packages/stream_chat_flutter/lib/src/message_list_view.dart

```diff
@@ -1199,10 +1199,12 @@ class _MessageListViewState extends State<MessageListView> {
       initialAlignment = _initialAlignment;
 
       WidgetsBinding.instance!.addPostFrameCallback((timeStamp) {
-        _scrollController?.jumpTo(
-          index: initialIndex,
-          alignment: initialAlignment,
-        );
+        if (_scrollController?.isAttached ?? false) {
```

Comment:
```suggestion
        if (_scrollController?.isAttached == true) {
```

consider this change please, I think it's cleaner